translate/storage/test_yaml.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+import sys
+from io import BytesIO
+from translate.storage import yaml, test_monolingual
+
+
+class TestYAMLResourceUnit(test_monolingual.TestMonolingualUnit):
+    UnitClass = yaml.YAMLUnit
+
+
+class TestYAMLResourceStore(test_monolingual.TestMonolingualUnit):
```

Comment:
Oops, I've based the tests on resx and the same issue is there (translate/storage/test_resx.py).